app/views/shared/_viewers_indicator.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="viewers_indicator">
   <% viewers = local_assigns[:data].viewers %>
-  <%= number_of_viewers(local_assigns[:data].userid, current_user.id, viewers) %>
+  <%= number_of_viewers(local_assigns[:data].userid, current_user.try(:id), viewers) %>
```

Comment:
Nice!

Reply:
Probably better as `current_user&.id` despite the ugly factor